proto/store/approval.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+option go_package = "generated-go/store";
+
+// IssuePayloadApproval is a part of the payload of an issue.
+// IssuePayloadApproval records the approval template used and the approval history.
+message IssuePayloadApproval {
+  ApprovalTemplate approval_template = 1;
```

Comment:
Could this be repeated in case we would match two approval templates in approval config?